helm/cluster-cloud-director/templates/cloud-provider-cloud-director-helmrelease.yaml

```
@@ -0,0 +1,52 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: {{ include "resource.default.name" $ }}-cloud-provider-cloud-director
+  namespace: {{ $.Release.Namespace }}
+  annotations:
+    cluster.giantswarm.io/description: "{{ .Values.clusterDescription }}"
+  labels:
+    cluster-apps-operator.giantswarm.io/watching: ""
+    giantswarm.io/service-priority: "{{ .Values.servicePriority }}"
+    {{- include "labels.common" . | nindent 4 }}
+    {{- range $key, $val := .Values.clusterLabels }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end}}
+spec:
+  suspend: true # It will be unsuspended by the post-install/post-upgrade hook in update-values-hook-job.yaml.
+  releaseName: cloud-provider-cloud-director
+  targetNamespace: kube-system
+  storageNamespace: kube-system
+  chart:
+    spec:
+      chart: cloud-provider-cloud-director
+      version: 0.2.0
+      sourceRef:
+        kind: HelmRepository
+        name: {{ include "resource.default.name" $ }}-default
+  kubeConfig:
+    secretRef:
+      name: {{ include "resource.default.name" $ }}-kubeconfig
+  interval: 10m
+  install:
+    remediation:
+      retries: 3
+  # Default values
+  # https://github.com/giantswarm/cloud-provider-cloud-director-app/blob/main/helm/cloud-provider-cloud-director/values.yaml
+  valuesFrom:
+    - kind: Secret
+      name: {{ .Values.userContext.secretRef.secretName | quote }}
+      valuesKey: "refreshToken"
+      targetPath: "global.basicAuthSecret.refreshToken"
+  values:
+    global:
+      basicAuthSecret:
+        create: true
+      vcdConfig:
+        clusterid: "" # It will be set by the post-install/post-upgrade hook in post-install-rdeid-job.yaml.
+        org: {{ .Values.cloudDirector.org | quote }}
+        ovdc: {{ .Values.cloudDirector.ovdc | quote }}
+        ovdcNetwork: {{ .Values.cloudDirector.ovdcNetwork | quote }}
+        site: {{ .Values.cloudDirector.site | quote }}
+        vAppName: {{ include "resource.default.name" $  | quote }}
+        vipSubnet: {{ .Values.network.loadBalancer.vipSubnet | quote }}
```

Depending on the decision around the [default](https://github.com/giantswarm/management-clusters-fleet/pull/393#discussion_r1118392799).
```suggestion
        vipSubnet: {{ .Values.network.loadBalancer.vipSubnet | quote }}
        enableVirtualServiceSharedIP: true
        oneArm:
          enabled: false
          startIP: "192.168.8.2"
          endIP: "192.168.8.100"
```